crates/contracts/artifacts/SushiSwapFactory.json

```diff
@@ -0,0 +1 @@
+IUniswapLikeFactory.json
```

Comment:
Note that this was previously linked to `UniswapV2Factory`. This was incorrect because it **did not** have the same bytecode, so if we started deploying SushiSwap factories in our E2E tests, the addresses retrieved by the `PairProvider` would have been wrong because the `INIT_CODE_DIGEST` would not match.